java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -315,4 +315,72 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
+
+    /**
+     * Displays a piece of generative computer art and the Redis version.<br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/lolwut/">redis.io</a> for details.
+     * @return Redis response.
```

Comment:
Consider reversing the sentence (since the version comes last): 
```
@return A piece of generative computer art along with the current Redis version. 
```

Reply:
Still just a couple instances of "@return Redis response" that need to be updated

Reply:
oh, thanks
Fixed